src/components/Banner/Banners_.ts

```diff
@@ -0,0 +1,95 @@
+import styled from "styled-components";
+import { Flex, Text } from "@radix-ui/themes";
+import theme from "../../theme";
+import { Link } from "react-router-dom";
+import { ExternalLinkIcon } from "@radix-ui/react-icons";
+
+const BaseBanner = styled(Flex)`
+  position: relative;
+  width: 100%;
+  min-height: 200px;
+  height: auto;
+  background-image: linear-gradient(
+    to bottom right,
+    #ff7e5f,
+    #feb47b,
+    #86a8e7,
+    #91eae4
+  );
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-start;
+  padding: 20px;
+  color: white;
+  border-radius: 20px;
+
+  @media (max-width: 768px) {
+    min-height: 150px;
+    padding: 15px;
+  }
+
+  @media (max-width: 480px) {
+    min-height: 120px;
+    padding: 10px;
+  }
+`;
+
+export const StyledOptionalLinkBanner = styled(BaseBanner)`
+  border-bottom-right-radius: 76px;
+
+  @media (max-width: 768px) {
+    border-bottom-right-radius: 50px;
+  }
+
+  @media (max-width: 480px) {
+    border-bottom-right-radius: 30px;
+  }
+`;
+
+export const StyledPromotedBanner = styled(BaseBanner)``;
+
+export const StyledFlex = styled(Flex)`
```

Comment:
In such cases I avoid creating custom components but apply these styles directly in Flex component
<Flex direction={'column} justify={'start} align={'start'} width={'100%'} >